examples/data/ship_specifications/PX121.json

```diff
@@ -52,7 +26,33 @@
 				"LOA": 82,
 				"BOA": 18,
 				"Depth": 8,
-				"APP": 2
+				"APP": 2,
```

Comment:
I am not so sure about putting all of this in hull.attributes... Apart from the weight calculation, the hull class was very clean and bottom-up. Design assumptions/criteria just don't fit in. (But where do they really fit in?)

I have been thinking for a while that we will be introducing more ways of specifying hulls, different parameterizations. Then one variant can simply be to specify some design attributes instead of calculating them. (Visualization will then have to show something very generic, like a transparent box.)

Reply:
Not everything I collected under `hull.attributes` are design assumptions, though. Some properties are actually descriptions of hull appendices. On the other hand, different ways to add design information/tags would come in hand. They could handle this characterization (does this hull have a bulb? what is the shape of the stern?) in a more structured way.